app/docs/cloud-instances/page.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from "@/components/ui/link"
+
+import { CloudInstancesTable } from "./cloud-instances-table"
+
+import { getInstances } from "@/lib/api/cloud-instances"
+
+export default async function CloudInstancesPage() {
+  const instances = await getInstances()
+
+  return (
+    <div className="flex flex-col gap-8">
+      <div className="flex flex-col gap-4">
+        <h1 className="text-2xl font-bold">Cloud Instances</h1>
+        <div className="flex flex-col gap-2">
+          <p>
+            This table helps you find the correct instance reference ID for use
+            in our <Link href="/api.html">API</Link>. Each row shows available
```

Comment:
Small note, don't believe we need to include the `.html` in these links, ie: https://ethproofs.org/api should work just fine

```suggestion
            in our <Link href="/api">API</Link>. Each row shows available
```